--- a/src/CMake/FindNetcdf.cmake	2023-09-03 12:44:08.000000000 -0600
+++ b/src/CMake/FindNetcdf.cmake	2023-09-03 12:44:20.000000000 -0600
@@ -20,12 +20,12 @@
 
 if (WIN32)
     if (MSVC_VERSION LESS "1910")
-      set(NETCDF_CXX_DIR ${NETCDF_DIR})
+      #set(NETCDF_CXX_DIR ${NETCDF_DIR})
       SET_UP_THIRD_PARTY(NETCDF_CXX LIBS netcdf_c++)
     endif()
     set(NETCDF_WIN32_DEFINES "DLL_NETCDF")
 else()
-    set(NETCDF_CXX_DIR ${NETCDF_DIR})
+    #set(NETCDF_CXX_DIR ${NETCDF_DIR})
     SET_UP_THIRD_PARTY(NETCDF_CXX LIBS netcdf_c++)
 endif()
 
